lib/debug/asserts.js

```diff
@@ -43,8 +43,23 @@ shaka.asserts.notImplemented = function() {};
 shaka.asserts.unreachable = function() {};
 
 
+shaka.asserts.patchAssert_ = function() {
+  var assert = console['assert'];
+
+  if (!assert) {
+    console.assert = function() {};
+  } else if (!assert.bind) {
```

Comment:
There is one here; I added it inside the if, but I can try to fit it on the end of the line, if that's more in-line with your style.